stdlib/public/TensorFlow/ShapedArray.swift

```diff
@@ -696,28 +722,39 @@ public extension ShapedArraySlice {
     return baseIndices.count
   }
 
+  /// The number of dimensions of the `ShapedArraySlice`.
   var rank: Int {
     return base.rank - indexingDepth
   }
 
+  /// The dimensions of the `ShapedArraySlice`.
```

Comment:
This can just be "shape" because it's pretty "well defined".

Reply:
Done in https://github.com/apple/swift/pull/23093/commits/3540dd5c23e763c7c29d5673f0db050580659617.